Dockerfile

```diff
@@ -23,6 +23,7 @@ RUN apk add --no-cache \
         py3-cryptography \
         py3-pip \
         py3-wheel \
+        xorriso \
```

Comment:
we will not need this package as we are not going to build template using ISO.

Reply:
well we do... thats how kickstart is being injected

Reply:
got it.